File: Selecao I/calc.cpp
/*@calc: Calculadora numérica*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    int n1, n2;
    char op;
    cin >> n1;
    cin >> n2;
    cin >> op;
    
    if(op == '+') {
        cout << n1 + n2 << endl;
    } else if(op == '-') {
        cout << n1 - n2 << endl;
    } else if(op == '*') {
        cout << n1 * n2 << endl;
    } else if(op == '/') {
        if(n2 != 0) {
            cout << n1 / n2 << endl;
        } else {
            cout << "Erro: Divisão por zero!";
        }
    } else {
        cout << "Operador inválido!";
    }
    return 0;
}

File: src/Selecao_I/Calc.java
/**
 * Questão: @calc: Calculadora numérica
 *
 * @author UxieGu1
 */
package Selecao_I;
import java.util.Scanner;

public class Calc {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n1 = scan.nextInt();
        int n2 = scan.nextInt();
        char op = scan.next().charAt(0);

        switch(op) {
            case '+':
                System.out.println(n1 + n2);
                break;
            case '-':
                System.out.println(n1 - n2);
                break;
            case '*':
                System.out.println(n1 * n2);
                break;
            case '/':
                if(n2 != 0) {
                    System.out.println(n1 / n2);
                } else {
                    System.out.println("Erro: Divisão por zero!");
                }
                break;
            default:
                System.out.println("Operador inválido!");
        }

        scan.close();
    }
}
